src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1490,6 +1490,82 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </Target>
 
   <!--
+    ====================================================================================
+                                        _GetProjectReferenceTargetFrameworkProperties
+
+    Builds the GetTargetFrameworkProperties target of all existent project references,
+    passing $(TargetFrameworkMoniker) as $(ReferringTargetFramework) and sets the
+    SetTargetFramework metadata of the project reference to the value that is returned.
+
+    This allows a cross-targeting project to select how it should be configured to
+    build against the most appropriate target for the referring target framework.
+
+    ======================================================================================
+  -->
+  <Target Name="_GetProjectReferenceTargetFrameworkProperties" Outputs="%(_MSBuildProjectReferenceExistent.Identity)">
```

Comment:
Or is it here to batch the target so the loose `%(Identity)` metadata below works?

---
In reply to: [78624878](https://github.com/Microsoft/msbuild/pull/1018#discussion_r78624878) [](ancestors = 78624878)